### ISSUE TITLE
TinaMarkdown doesn't use alt tags from markdown.

### ISSUE THREAD

**Issue by perkinsjr** [tinacms/tinacms#2980]
When using `TinaMarkdown` alt text isn't inserted into the image tag even if it is included:

Example
```![example image](/img/example.jpg)```

returns 

`<a src="/img/example.jpg")`

We should include the alt text for images when supplied.

**Comment by burntcustard** [comment]
Not passing alt text to `<img>` elements when using `TinaMarkdown` is still a bug and in my opinion shouldn't have been marked as wontfix.

I believe I've tracked down at least half of the cause - [line 269 in `packages/tinacms/src/rich-text/index.tsx`](https://github.com/tinacms/tinacms/blob/50f86caed86fbc6526586a11cf66e70504c6972f/packages/tinacms/src/rich-text/index.tsx#L269) is:
```
return <img src={child.url} alt={child.caption} />
```

It's attempting to put the `caption` field as alt text, rather than the alt text field.

I've been able to get around this locally in two ways, either:

**a) Use a markdown image `title`, which gets pulled through as the caption and then is set as alt text:**

```
![Alt that doesn't work](/path/to/img.jpg "image title that's set as alt text")
```

I don't like this solution, because it's generally not how alt text is done in markdown, as far as I've seen isn't documented anywhere on the [Tina Docs](https://tina.io/docs/), and feels fragile if/when this alt text issue is "fixed". It _is_ simple though.

**b) By changing that (via line 153 in `/node_modules/tinacms/dist/rich-text/index.js`) to:**

```
return <img src={child.url} alt={child.caption || child.alt} />
```

My thought here is that if either of those are set, it will use them for the alt text of the image. Still slightly confusing, but as far as I can tell it's doesn't really break anything in my Tina projects, and wouldn't in any others either.

Unfortunately with that "fix", I see what I believe is a related console error in the admin view, because of the alt text in my markdown confusing Tinacms' inner workings somehow:
```
Warning: Extra attributes from the server: alt
img
[...]
```

Hopefully at least one of these suggestions helps out anyone who stumbles across this issue after having the same problem themselves!

**Comment by burntcustard** [comment]
Almost 3 months on and this seems to still be a major accessibility / TinaCMS UX issue.

As a bit of extra info, it's not just an issue when editing markdown files, but when using the editor as well, the image `Alt` field doesn't do anything, and the `Caption` field sets the `alt` attribute of the `<img>` element.

**Comment by EthanShoeDev** [comment]
I patched the code so that alt tags work.
You can use 
```json
{
  "dependencies": {
    "@tinacms/datalayer": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/@tinacms/datalayer/tinacms-datalayer-1.2.39.tgz",
    "next-tinacms-s3": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/next-tinacms-s3/next-tinacms-s3-4.0.0.tgz",
    "tinacms": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/tinacms/tinacms-2.1.0.tgz",
    "tinacms-authjs": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/tinacms-authjs/tinacms-authjs-4.0.0.tgz",
    "tinacms-gitprovider-github": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/tinacms-gitprovider-github/tinacms-gitprovider-github-1.0.10.tgz",
    "@tinacms/cli": "https://github.com/EthanShoeDev/tinacms/raw/patched-version/packages/@tinacms/cli/tinacms-cli-1.5.52.tgz"
  }
}
```
To use the patched versions.

You may have to use `npm i --legacy-peer-deps`
